### DISCUSSION TITLE
The web3Modal wallet cannot detect the plugin wallets installed on my website, such as phantom, metaMask, Trust and other wallets

### DISCUSSION THREAD

**Discussion by maicFir** [reown-com/appkit#2516] [Q&A]
I found that I integrated @web3modal/solana and @solana/web3.js in my website, and the web3Modal wallet could not detect the plugin wallet installed on my website, and the MetaMask wallet could not establish a wallet connection by scanning the code

![Snipaste_2024-06-22_14-30-18](https://github.com/WalletConnect/walletconnect-monorepo/assets/21973334/6ac4bf6a-4b69-4ec6-b079-f699a6d3610a)

![Snipaste_2024-06-22_14-31-55](https://github.com/WalletConnect/walletconnect-monorepo/assets/21973334/defb9c27-0a45-4325-a5f7-91eaeb375370)
my dependencies package about "@solana/web3.js": "^1.90.0"、"@web3modal/solana": "5.0.2"

import { createWeb3Modal, defaultSolanaConfig } from "@web3modal/solana/react";
import { solana, solanaTestnet, solanaDevnet } from "@web3modal/solana/chains";
const chains = [solana];

const projectId: any = process.env.NEXT_PUBLIC_PROJECT_ID;
const solanaConfig = defaultSolanaConfig({
  metadata: {
    name: "xxx",
    description: "xx",
    url: "xx", // origin must match your domain & subdomain
    icons: ["/images/logo/logo.png"],
  },
  chains,
  projectId,
});

export const web3Modal = createWeb3Modal({
  solanaConfig,
  chains,
  projectId,
  featuredWalletIds: [
    "c57ca95b47569778a828d19178114f4db188b89b763c899ba0be274e97267d96",
    "a797aa35c0fadbfc1a53e7f675162ed5226968b44a19ee3d24385c64d1d3c393",
    "971e689d0a5be527bac79629b4ee9b925e82208e5168b733496a09c0faed0709",
    "8a0ee50d1f22f6651afcae7eb4253e52a3310b90af5daef78a8c4929a9bb99d4",
    "4622a2b2d6af1c9844944291e5e7351a6aa24cd7b23099efac1b2fd875da31a0",
    "fd20dc426fb37566d803205b19bbc1d4096b248ac04548e3cfb6b3a38bd033aa",
    "1ae92b26df02f0abca6304df07debccd18262fdf5fe82daa81593582dac9a369",
    "38f5d18bd8522c244bdd70cb4a68e0e718865155811c043f052fb9f1c51de662",
  ],
});

**Comment by rtomas** [comment]
Could you try to update `@web3modal/solana` to latest version ?
